samples/net/lwm2m_client/README.rst

```diff
@@ -90,7 +93,7 @@ To build the lwm2m-client sample for QEMU with DTLS support do the following:
    :zephyr-app: samples/net/lwm2m_client
    :host-os: unix
    :board: qemu_x86
-   :conf: prj_dtls.conf
+   :conf: prj.conf overlay-dtls.conf
```

Comment:
I assume it was tested that the rendered output will be correct (have doublequotes around both conf files mentioned, etc.)

Reply:
Hm.. good point.  @dbkinder is there a way to see how this will render?

Reply:
You can generate documentation locally on your computer following the instructions in http://docs.zephyrproject.org/README.html

It will look like this:

    cd $ZEPHYR_BASE/samples/net/lwm2m_client
    mkdir build && cd build
    cmake -GNinja -DBOARD=qemu_x86 -DCONF_FILE=prj_dtls.conf overlay-dtls.conf ..
    ninja run

If you change the ``:conf:`` parameter to use quotes in the .rst file, like so:

    :conf: "prj.conf overlay-dtls.conf"

the resulting output would be:

    cmake -GNinja -DBOARD=qemu_x86 -DCONF_FILE="prj_dtls.conf overlay-dtls.conf" ..